src/display/annotation_layer.js

```diff
@@ -678,6 +688,12 @@ class ChoiceWidgetAnnotationElement extends WidgetAnnotationElement {
       selectElement.appendChild(optionElement);
     }
 
+    selectElement.addEventListener("change", function (event) {
```

Comment:
Shouldn't we also use `storage.getOrCreateValue` in this function to select the value in the storage a few lines above?